targets/ramips-mt7621

```diff
@@ -8,8 +8,12 @@ device d-link-dir-860l-b1 dir-860l-b1
 
 ## ZBT
 
-device zbt-wg3526 zbt-wg3526
+device zbt-wg3526-16m zbt-wg3526-16M
 factory
+manifest_alias zbt-wg3526
+device zbt-wg3526-32m zbt-wg3526-32M
+factory
+manifest_alias digineo-ac1200-pro
```

Comment:
please remove the digineo alias ;-)
we never had that device in gluon with this name, therefore we need no other name than the one LEDE/OpenWrt uses.

Reply:
:-1: this device is shipped with lede (where maybe old lede name is still `ac1200pro`)

Reply:
@genofire Lede does not check on the image name. This is only for the Gluon autoupdater. I assumed as there is no factory for that router that it would be nice if people could still find their device by name, but I think that is the wrong way. It would be better if the Gluon makefiles could make symlinks in the factory dir pointing to the sysupgrade-files. Then it would make sense to create such an alias...